src/app/pages/visitsCalendar/visitsCalendar.component.html

```diff
@@ -43,22 +25,28 @@
 
 <modal #visitModal>
     <modal-header [show-close]="true">
-        <h1 class="modal-title" >Visit</h1>
+        <h1 class="modal-title">Visit</h1>
 
     </modal-header>
     <modal-body>
         <h2>Doctor {{doctor?.firstName}} {{doctor?.lastName}}</h2>
         <h2>{{modalDate}}, {{modalTime}}</h2>
         <h3>Patient's data</h3>
         <!--<div class="dropdown">-->
-            <!--<button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{{patientName}}-->
-                <!--<span class="caret"></span></button>-->
-            <ul >
+        <!--<button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{{patientName}}-->
+        <!--<span class="caret"></span></button>-->
+        <div *ngIf='currentSlotTaken'>
+            <a (click)="showPatientDetails()">
+    Taken by: {{patient.account.personalDetails.firstName}} {{patient.account.personalDetails.lastName}}</a>
```

Comment:
https://stackoverflow.com/questions/37385535/calling-method-from-a-angular-2-class-inside-template
niby takie proste ale to kolejna przekombinowana składnia której nie jestem w stanie zmusić do działania :/